syntax = "proto3";

package Serialization;

import "Bus.proto";
import "Stop.proto";
import "RoutingSettings.proto";
import "RenderSettings.proto";
import "database.proto";

message TransportCatalog
{
  repeated Stop stops = 1;
  repeated Bus buses = 2;
  RoutingSettings routing_settings = 3;
  RenderSettings render_settings = 4;
  YellowPages.Database yellow_pages_database = 5;
}